src/css/imports/mixins.less

```diff
@@ -363,6 +363,10 @@
           .jw-text-countdown {
               height: auto;
               line-height: normal;
+
+              &.jw-icon-rewind {
```

Comment:
This looks too specific. Is there a `.jw-icon-rewind` class definition higher up you can add this too (like where the :before rule is defined for the icon)?